src/services/completions.ts

```diff
@@ -1885,7 +1905,16 @@ namespace ts.Completions {
 
                 let existingName: __String | undefined;
 
-                if (isBindingElement(m) && m.propertyName) {
+                if (isSpreadAssignment(m)) {
+                    const expression = m.expression;
+                    const symbol = typeChecker.getSymbolAtLocation(expression);
+                    const type = symbol && typeChecker.getTypeOfSymbolAtLocation(symbol, expression);
+                    const properties = type && (<ObjectType>type).properties;
+                    if (properties) {
+                        membersDeclaredBySpreadAssignment.push(...properties);

```

Comment:
Actually make this a map so that later you can just check if membersDeclaredBySpreadAssignment.has(filteredSymbol.name) instead of having to loop through all the symbols later